src/cmr/concepts/orderOption.js

```diff
@@ -0,0 +1,42 @@
+import Concept from './concept'
+
+export default class OrderOption extends Concept {
+  /**
+   * Instantiates an OrderOption object
+   * @param {Object} headers HTTP headers provided by the query
+   * @param {Object} requestInfo Parsed data pertaining to the Graph query
+   * @param {Object} params GraphQL query parameters
+   */
+  constructor(headers, requestInfo, params) {
+    // This concept uses the "-" character to delineate spaces in CMR we must pass it
+    // in this form to fetch order option concepts from CMR
+    super('order-options', headers, requestInfo, params)
```

Comment:
Should order-options be singular?

Reply:
In this case it has to be plural because it's going off to` cmr/search/order-options`